docs/gitops-automation.md

```diff
@@ -9,74 +9,148 @@ This is the configuration that defines the flow that happens when an update happ
 
 ## What is the GitOps Automation Configuration?
 
-The GitOps Automation configuration consists of two types of configuration: `apps` and `targets`.
-Fundamentally, GitOps is connecting `apps` with `targets` by automatically deploying the app into the target.
+The GitOps Automation configuration consists of two types of configuration: `apps` and `clusters`.
+Fundamentally, GitOps is connecting `apps` with `clusters` by automatically deploying the app into the cluster.
 
 ## Where does it live?
 
-Weave GitOps supports three places to store this configuration:
-
-1. In the same repository as your workload definition (e.g. in the same repository containing your k8s YAMLs)
-2. In a separate repository that can cover different applications.
-3. Only configured into the cluster (not stored)
-
-The default behaviour is #1, which is to store the GitOps Automation configuration in the same repository as your workload YAMLs are stored.
-This is the best option for a team that manages their own app and infrastructure and wants a simple, easy deployment of Weave GitOps.
-
-Option 2 is designed for scenarios where there are multiple applications managed by a single team (maybe a platform ops team or a team that owns more than one app).
-
-Option 3 is a basic approach that is useful if you are just learning about GitOps and don't want to be able to change the automation flow.
+The configuration is stored in the repository specified in `gitops install`. The application manifests can be stored in the same repository as the configuration or a separate repository.
 
 ## What files are there
```

Comment:
nit - should have question mark given above

Reply:
:man_facepalming:

Reply:
I can't find this text anywhere, so I assume it was fixed. Marking as resolved.